specification/compute/resource-manager/Microsoft.Compute/preview/2021-06-01-preview/diagnostic.json

```diff
@@ -206,6 +206,98 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/diagnostics/diskInspection/registerStorageConfiguration": {
+      "post": {
```

Comment:
using this POST model means that users will not be able to set up their subscription completely using just a template deployment. That means they will have to perform imperative calls via portal, powershell, cli, or similar which is something that should typically be avoided for configuration like this.